### PR TITLE
ES: Add Euronews.

### PR DESCRIPTION
**Pull request by heroku-miraheze**

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
Euronews is a French channel: https://www.lyngsat.com/tvchannels/fr/EuroNews-Espanol.html

**Comment by heroku-miraheze** [comment]
Moved to fr.m3u

File: channels/fr.m3u [hunk 1]
```diff
@@ -145,6 +145,8 @@ http://51.195.88.12:4050/live/euronewsmobpp2/playlist.m3u8
 https://rakuten-euronews-5-de.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="EuronewsenEspanol.fr" tvg-name="Euronews en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/K6bo1R3.jpg" group-title="News",Euronews en Español (720p)
 https://rakuten-euronews-4-es.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="EuronewsEnEspanol.fr" tvg-name="Euronews en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/K6bo1R3.jpg" group-title="News",Euronews en Español (YouTube)
```
Comment (freearhey) [inline channels/fr.m3u:148]: There is no need to specify the source of broadcast in the name of the channel.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/fr.m3u (hunk 1 above)
Comment (heroku-miraheze) [reply channels/fr.m3u:148]: Fixed.

Comment (heroku-miraheze) [reply channels/fr.m3u]: Fixed.

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
Thanks!

**Pull request merged**